specs-go/state.go

```diff
@@ -33,3 +33,19 @@ type State struct {
 	// Annotations are key values associated with the container.
 	Annotations map[string]string `json:"annotations,omitempty"`
 }
+
+type SeccompState struct {
+	// Version is the version of the specification that is supported.
+	Version string `json:"ociVersion"`
+	// SeccompFd is the file descriptor for Seccomp User Notification
```

Comment:
```suggestion
	// SeccompFd is the index of the file descriptor in the
	// `SCM_RIGHTS` array referring to the seccomp notify file
	// descriptor. It is always zero (the first entry in the array).
```

Reply:
Oh, nice catch. I forgot to update these :see_no_evil:.

Fixed, thanks!